src/DeviceListener.js

```diff
@@ -78,6 +78,8 @@ export default class DeviceListener {
         this._dismissedThisDeviceToast = false;
         this._keyBackupInfo = null;
         this._keyBackupFetchedAt = null;
+        this._ourDeviceIdsAtStart = null;
+        this._displayingToastsForDeviceIds = new Set();
```

Comment:
Is this really the only changes to device listener...? (Just making sure something wasn't left behind locally.)

Reply:
yeah, I think it was that it just wasn't clearing the old data